packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -190,18 +188,13 @@ contract OptimisticGovernor is Module, Lockable {
      */
     function proposeTransactions(Transaction[] memory _transactions, bytes memory _explanation) external nonReentrant {
         // note: Optional explanation explains the intent of the transactions to make comprehension easier.
-        uint256 id = prevProposalId + 1;
-        prevProposalId = id;
         uint256 time = getCurrentTime();
         address proposer = msg.sender;
 
         // Create proposal in memory to emit in an event.
         Proposal memory proposal;
         proposal.requestTime = time;
 
-        // Construct the ancillary data.
-        bytes memory ancillaryData = AncillaryData.appendKeyValueUint("", "id", id);
```

Comment:
Instead of dropping ancillary data, should we, instead, append the proposal hash? Without this, you get a weird problem: you can't make two proposals in the same block, since the requests would be identical (same timestamp and ancillary data) regardless of the transactions. Not a huge issue, but would be confusing for a user. It also just seems more comforting for a voter to have some identifying information in the ancillary data, so it's super obvious what they're voting on.